Fix Bone Shape of Skeleton Deformation affected by Transformations

rodolforg opened the pull request:
Fix #2526

morevnaproject commented:
Hi! Apologies for a long delay. I have tested this PR and it works great!

Just found one issue with this file - 
[skeleton-deform.zip](https://github.com/synfig/synfig/files/8154163/skeleton-deform.zip)

It shows bones like this -
![screenshot_002](https://user-images.githubusercontent.com/332868/156003272-c3ff7ca0-9adf-4267-bd03-06f0bbfa3c6f.png)

...and here is the same file when opened in Synfig without this PR applied -
![screenshot_003](https://user-images.githubusercontent.com/332868/156003366-29eb2daa-f0aa-4403-91e8-55c0e0dc7eb9.png)

rodolforg commented:
> Just found one issue with this file -

Fixed. Added a single line (#160):
```c++
cr->move_to(shape.p0[0]+shape.r0*cos(angle1), shape.p0[1]+shape.r0*sin(angle1));
```

morevnaproject reviewed (approved):
works perfectly now! Thank you so much!
